src/library/button/button.tsx

```diff
@@ -27,11 +27,17 @@ export interface IButtonProps {
     onPressIn?: any;
     onPressOut?: any;
     bottomLabel?: string;
+    testIDWrapper?: string;
```

Comment:
I think we should enforce this. It will force us to set testId for all future screens 
* in a future separate PR :)